tests/sparsity/dynamic_sparsity_pattern_iterator_07.cc

```diff
@@ -0,0 +1,77 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2004 - 2017 by the deal.II authors
```

Comment:
Do we adjust dates?

Reply:
We have a script that will add 2018 next time we run it. The dates are not incorrect: I cloned an existing test, so the new tests' copyright extends back to these older tests.